.github/workflows/gem-push.yml

```diff
@@ -7,7 +7,7 @@ on:
 jobs:
   build:
     name: Build + Publish
-    runs-on: macos-latest
+    runs-on: macos-11
```

Comment:
Latest is not actually latest, but 10.15. This ensures we use 11 which is the first that comes with proper M1 support.